frontend/packages/kubevirt-plugin/integration-tests/tests/utils/constants/vm.ts

```diff
@@ -50,11 +50,7 @@ export enum VM_STATUS {
 // Network
 export enum NIC_MODEL {
   VirtIO = 'VirtIO',
-  e1000 = 'e1000',
```

Comment:
I suppose we don't need it for testing, but it doesn't hurt to have it here

Reply:
ok